.github/workflows/build.yml

```diff
@@ -142,3 +142,20 @@ jobs:
           cd port/linux
           make DEBUG=1 SECURE=1 IPV4=1 TCP=1 PKI=1 DYNAMIC=1 CLOUD=1 JAVA=1 IDD=1 
 
+
+   make_linux_swupdate:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE,  so your job can access it
+      - uses: actions/checkout@v2
+      # Runs a set of commands using the runners shell
+      - name: make secured with software update (using boost)
+        run: |
+          # the setup
+          #sudo apt-get -y install libboost-all-dev
```

Comment:
This is supposed to be commented out, right?

Reply:
yes, I have removed boost, now using regex to do the url validation, avoiding the linking with boost